CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.14.0 (May 22, 2020)
+
+- **[Feature]**: Update of iOS SDK to `2.17.0` and Android SDK to `2.17.1`
```

Comment:
Should it be 0.13.1? Or why we changed the version to 0.14.0?

Reply:
Since 2.17.x changed some native behaviours and deprecated some methods (https://tokbox.com/developer/sdks/android/release-notes.html), I would bump the version to `0.14.0` but it's a not a big deal. 👍

Reply:
Fair enough. Thanks!